modules/web-console/docker/standalone/Dockerfile

```diff
@@ -63,22 +45,22 @@ WORKDIR /opt/web-console
 COPY build .
 
 # Install node modules.
-RUN cd /opt/web-console/frontend && npm install --no-optional --prod && npm run build
-RUN cd /opt/web-console/backend && npm install --only=production --no-optional
+RUN cd /opt/web-console/frontend && \
+    npm install --no-optional --prod && \
+    npm run build
 
-# Returns to base path.
-WORKDIR /opt/web-console
+RUN cd /opt/web-console/backend && \
+    npm install --only=production --no-optional
```

Comment:
Please, use the same command format as before (npm install --no-optional --prod).